packages/menu/src/presenters/stylesheet.js

```diff
@@ -64,6 +65,12 @@ export default function stylesheet(props, themeData) {
             transition: `background-color 0.3s cubic-bezier(0.4,0,0.2,1)`
           }
         : {}),
+      ...(isPressed && selected && !unselect
+        ? {
+            backgroundColor: themeData[`menu.item.hover.backgroundColor`],
+            cursor: `default`
```

Comment:
I don't think this is where you want to set the `cursor` back to default as this is only during the `pressed` state, so currently when the Option is selected and you hover over it, it looks like you can still click and when you press the cursor changes giving it an odd effect.